rabbitmq/rabbitmq.go

```diff
@@ -352,16 +419,20 @@ func (client *DefaultClient) publishToLndhubExchange(ctx context.Context, invoic
 		},
 	)
 	if err != nil {
-		captureErr(client.logger, err)
 		return err
 	}
 
-	client.logger.Debugf("Successfully published invoice to rabbitmq with RHash %s", invoice.RHash)
+	client.logger.Infoj(log.JSON{
+		"subroutine":   "invoice publisher",
+		"message":      "succesfully published invoice",
+		"payment_hash": invoice.RHash,
+	})
 
 	return nil
 }
 
-func captureErr(logger *lecho.Logger, err error) {
-	logger.Error(err)
+func captureErr(logger *lecho.Logger, err error, j log.JSON) {
+	j["error"] = err
+	logger.Errorj(j)
 	sentry.CaptureException(err)
```

Comment:
when I currently look at the sentry stack traces it does not give much information and it does not contain the original stack trace but links to this `CaptureException` error

I read here something about a different erros package and wrapping an error to keep the stack trace? 
https://incident.io/blog/golang-errors
does that make sense?

Reply:
Go stacktraces inherently require more work, yes. So that would be something to add later.